node_modules/gh-config/lib/api.js

```diff
@@ -230,12 +230,20 @@ var updateConfig = module.exports.updateConfig = function(ctx, appId, update, ca
     validator.check(_.keys(update).length, {'code': 400, 'msg': 'At least one config field needs to be specified'}).min(1);
     _.each(update, function(val, key) {
         validator.check(key, {'code': 400, 'msg': 'Unknown config field'}).isIn(ConfigDAO.EDITABLE_FIELDS);
+
         // Config fields that start with `allow` or `enable` should be boolean values
         if (key.indexOf('allow') === 0 || key.indexOf('enable') === 0) {
             val = GrasshopperUtil.getBooleanParam(val);
             validator.check(null, {'code': 400, 'msg': 'Boolean value expected for ' + key}).isBoolean(val);
             update[key] = val;
         }
+
+        // A valid integer must be provided for the `academic year` config field
+        if (key === 'academicYear') {
+            val = GrasshopperUtil.getNumberParam(val);
+            validator.check(val, {'code': 400, 'msg': 'Integer value expected for ' + key}).isInt();
+            update[key] = val;
```

Comment:
It might be useful for other apps as well. As this can only be entered by administrators I don't think we need to put more reasoning behind it